### PR TITLE
add PipelineImageTransform docs

### PR DESCRIPTION
**Pull request by kepricon**
https://github.com/deeplearning4j/DataVec/issues/292

updated https://deeplearning4j.org/simple-image-load-transform to reflect the use of PipelineImageTransform.

/cc @huitseeker

### CODE REVIEW

File: simple-image-load-transform.md [hunk 1]
```diff
@@ -93,29 +93,61 @@ You can chain transformations as shown below, write your own classes that will a
...
 recordReader.initialize(trainData,transform);
 ~~~
 
-## JavaCV , OpenCV and ffmpeg Filters
 
-ffmpeg and OpenCV provide open source libraries for filtering and transforming images and video. Access to ffmpeg filters in versions 7.2 and above is available by adding the following to your pom.xml file, replacing the version with the current version. 
+
+## JavaCV , OpenCV and FFmpeg Filters
+
+FFmpeg and OpenCV provide open source libraries for filtering and transforming images and video. Access to ffmpeg filters in versions 7.2 and above is available by adding the following to your pom.xml file, replacing the version with the current version.
```
Comment (huitseeker) [inline simple-image-load-transform.md]: This ffmpeg should be FFmpeg

File: simple-image-load-transform.md [hunk 2]
```diff
@@ -93,29 +93,61 @@ You can chain transformations as shown below, write your own classes that will a
...
 
 ~~~java
 recordReader.initialize(trainData,transform);
 ~~~
 
-## JavaCV , OpenCV and ffmpeg Filters
 
-ffmpeg and OpenCV provide open source libraries for filtering and transforming images and video. Access to ffmpeg filters in versions 7.2 and above is available by adding the following to your pom.xml file, replacing the version with the current version. 
+
+## JavaCV , OpenCV and FFmpeg Filters
```
Comment (huitseeker) [inline simple-image-load-transform.md]: JavaCV, OpenCV ...

File: simple-image-load-transform.md [hunk 3]
```diff
@@ -93,29 +93,61 @@ You can chain transformations as shown below, write your own classes that will a
...
 * [JavaCV](https://github.com/bytedeco/javacv)
 * [OpenCV](http://opencv.org/)
-* [ffmpeg](http://ffmpeg.org/)
+* [FFmpeg](http://ffmpeg.org/)
 
 
 
 ## Handing off to fit
-dl4j's neural net's take either a dataset or a dataset iterator to fit too. These are fundamental concepts for our framework. Please refer to other examples for how to use an iterator. Here is how you contruct a dataset iterator from an image record reader.
+Deeplearning4j's neural net's take either a dataset or a dataset iterator to fit too. These are fundamental concepts for our framework. Please refer to other examples for how to use an iterator. Here is how you contruct a dataset iterator from an image record reader.
```
Comment (huitseeker) [inline simple-image-load-transform.md]: Deeplearning4j's neural nets take either

File: simple-image-load-transform.md [hunk 4]
```diff
@@ -93,29 +93,61 @@ You can chain transformations as shown below, write your own classes that will a
...
+ImageTransform flip = new FlipImageTransform();
+
+List<Pair<ImageTransform, Double>> pipeline = new LinkedList<>();
+pipeline.add(new Pair<>(randCrop, 1.0));
+pipeline.add(new Pair<>(flip, 0.5));
+
+ImageTransform transform = new PipelineImageTransform(pipeline, shuffle);
+```
+The pipeline can also be randomly shuffled with each transform in order to increase the available dataset.  
+If you set `shuffle` as true, you can get a image pipelines that randomly execute.
```
Comment (huitseeker) [inline simple-image-load-transform.md]: If you set `shuffle` to true, you can get an image pipeline that executes each transformation at random.

**Comment by kepricon** [comment]
Thank you very much for this review :) @huitseeker 

I applied your comments and will merge it shortly.

**Pull request merged**